Model/PaymentMethod.php

```diff
@@ -90,14 +90,26 @@ public function getConfig()
             $this->_oPayment->oxpayments__wdoxidee_httppass->value
         );
 
+        self::_addAdditionalConfigInfo($oConfig);
```

Comment:
Please rename it to _addAnalyticsShopInfo()

Reply:
👍